OpenEdXMobile/res/values/dimens.xml

```diff
@@ -166,6 +170,7 @@
     <!-- Stray dimensions -->
     <dimen name="notification_settings_margin">12dp</dimen>
     <dimen name="fullscreen_message_max_width">240dp</dimen>
+    <dimen name="divider_with_text_margin">12dp</dimen>
```

Comment:
Those "divider" margins specific to the padding above and below the divers, not within the divider itself. I'll move it, but I think the necessity to point this out is debateable.